src/modules/unit/components/MapView.js

```diff
@@ -76,17 +76,16 @@ class MapView extends Component {
   }
 
   centerMapToUnit = (unit: Object) => {
-    if (this.state.isMobile) {
-      const location = getUnitPosition(unit);
-      location[0] += 0.035;
-      location[1] -= 0.005;
-      // For some reason could not use reverse here so had to do this weird way.
-      this.refs.map.leafletElement.setView(location, DEFAULT_ZOOM);
+    const location = getUnitPosition(unit);
+    if (!this.state.isMobile) {
+      this.refs.map.leafletElement.setView(location)
     } else {
-      const location = getUnitPosition(unit);
-      location[1] -= 0.07;
-
-      this.refs.map.leafletElement.setView(location, DEFAULT_ZOOM);
+      // On mobile we want to move the map 250px down from the center, so the
+      // awkwardly big info box does not hide the selected unit.
```

Comment:
I understand that no harm was meant here, but I am going to remove _awkwardly_ from this comment in order to avoid commenting on a design choice somewhere where designers are not going to be able to respond.

I'm happy to forward any design feedback you may have.